esp/src/src-react/components/Logs.tsx

```diff
@@ -137,7 +137,11 @@ export const Logs: React.FunctionComponent<LogsProps> = ({
                 formatter: ts => {
                     if (ts) {
                         if (ts.indexOf(":") < 0) {
-                            return timestampToDate(ts).toISOString();
+                            const date = timestampToDate(ts);
+                            if (date.toString() !== "Invalid Date") {
```

Comment:
I think the correct way to test if a JS Date is valid is `!isNaN(date)` or even better `date instanceof Date && !isNaN(date)`?

Reply:
new Date("asdf") is an instanceof Date, and TypeScript wouldn't allow just isNaN(date), so had to add the .getTime()